### DISCUSSION TITLE
Firebase SDK v9

### DISCUSSION THREAD

**Discussion by rskinnerc** [nuxt-community/firebase-module#519] [Ideas]
Hello.

I just realized the v9 beta release of Firebase sdk is here

https://firebase.google.com/docs/reference/js/v9

I'm wondering whether it is supported with the Firebase Module for Nuxt, and if it is not, is there any plan to support if soon?

Thank you

**Comment by lupas** [comment]
Hi @rskinnerc 

See the thread and my answer here:
https://github.com/nuxt-community/firebase-module/issues/531
